packages/composer/amazeelabs/silverback_iframe_theme/silverback_iframe_theme.info.yml

```diff
@@ -0,0 +1,37 @@
+name: Silverback Iframe Theme
+type: theme
+base theme: false
+description: 'Theme for amazeelabs/silverback_iframe module.'
+package: Silverback
+core_version_requirement: 9.x
+
+libraries:
+  - silverback_iframe_theme/global
+libraries-override:
+  commerce_order/total_summary:
+    css:
+      layout:
+        css/commerce_order.total_summary.css: false
+  commerce_order/address_book:
+    css:
+      layout:
+        css/commerce_order.address_book.css: false
+  commerce_order/form:
+    css:
+      layout:
+        css/commerce_order.module.css: false
+  commerce_checkout/checkout_progress:
+    css:
+      theme:
+        css/commerce_checkout.progress.css: false
+  commerce_checkout/form:
+    css:
+      theme:
+        css/commerce_checkout.form.css: false
+  address/form:
+    css:
+      theme:
+        css/address.form.css: false
```

Comment:
Those excludes feel arbitrary. I think we should either remove the entire CSS layer (probably `theme` and `layout`?) entirely, or we leave these untouched and leave it to the projects sub-theme to exclude unused styles. In cases where the embedded pages are not central to the actual product, we might actually want these styles and just slightly adjust them.

Reply:
I just copied this part over 🙈 

Project's sub-theme sounds great. Because we still need to add project-specific CSS.

I'll remove the overrides in this PR to not have breaking changes later.

Reply:
But, wait, how do we know them which theme to enable when `iframe=true` is in URL?

Reply:
Addressed in https://github.com/AmazeeLabs/silverback-mono/pull/793/commits/970d5e9ed02ea2d8537c83fe81ae062f6062e86f